docs/monitoring/spm-faq.md

```diff
@@ -297,37 +298,37 @@ This approach is not encouraged because of the obvious security
     options are not possible. As usual, restart spm-monitor after this
     change.
 
-### When should I run Standalone and when Embedded SPM monitor?
+### When should I run Standalone and when Embedded App Agent?
 
-[Standalone SPM monitor](spm-monitor-standalone)
+[Standalone App Agent](spm-monitor-standalone)
 runs as a separate process, while the [Embedded monitor](spm-monitor-javaagent) runs embedded in the
 Java/JVM process. Thus, if you are monitoring a non-Java application,
 Standalone monitor is the only option. Standalone monitor is a bit more
 complex to set up when one uses it to monitor Java applications because
 it typically requires one to enable out-of-process JMX access, as
-described on [Standalone SPM monitor page](spm-monitor-standalone). With Embedded monitor
-this is not needed, but one needs to add the SPM agent to the Java
+described on [Standalone App Agent page](spm-monitor-standalone). With Embedded monitor
+this is not needed, but one needs to add the Sematext Agent to the Java
 command-line and restart the process of the monitored application. When
-running Standalone monitor one can update the SPM monitor without
+running Standalone monitor one can update the App Agent without
 restarting the Java process being monitored, while a restart is needed
-when the Embedded SPM monitor is being used. To be able to [trace transactions](/tracing) 
+when the Embedded App Agent is being used. To be able to [trace transactions](/tracing)
 or [database operations](/tracing/database-operations) you need to use the
-Embedded SPM monitor.
+Embedded App Agent.
 
 ### Can I use Sematext for (business) transaction tracing?
 
 Yes, see [Transaction Tracing](/tracing).
 
-### Can I move SPM client to a different directory?
+### Can I move Sematext Agent to a different directory?
 
 Yes. Soft move script moves all SPM files/directories to a new
```

Comment:
Should probably get rid of various occurrences of "SPM" in this FAQ, too, at this time.